mkosi/burn.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import os
+import sys
+
+from mkosi.config import (
+    MkosiArgs,
+    MkosiConfig,
+    OutputFormat,
+)
+from mkosi.log import die
+from mkosi.run import run
+
+def run_burn(args: MkosiArgs, config: MkosiConfig) -> None:
+
+    if config.output_format not in (
+        OutputFormat.disk,
+        OutputFormat.esp,
+    ):
```

Comment:
That's short enough to fit on a line within our line length limit.
```suggestion
    if config.output_format not in (OutputFormat.disk, OutputFormat.esp):
```